scripts/measurements/darshini-test.sh

```diff
@@ -28,7 +28,7 @@ do
     echo -e "\n\n"  >> darshini-test.log
     #run one experiment in Darshini
     curl -H "Content-Type: application/json" -d '{"email": "abc", "password": "abc"}' http://localhost:8080/protocolanalyzer/signin
-    curl -X GET -H "Content-Type: application/json" http://localhost:8080/protocolanalyzer/test?pcapPath=\'$pcapFile\'
+    curl -X GET -H "Content-Type: application/json" http://localhost:8080/protocolanalyzer/test?pcapPath=$pcapFile
```

Comment:
the single quotes protect files names with spaces and special characters in them. It might be better to keep them.

Reply:
again, the log files must be put in BITS-Darshini/data/log directory.

Reply:
@prasadtalasila the script with single quotes is not working properly.